src/main/java/me/night0721/lilase/events/ChatReceivedEvent.java

```diff
@@ -96,10 +97,12 @@ public void onChat(ClientChatReceivedEvent event) {
                 Flipper.state = FlipperState.NONE;
                 Lilase.cofl.toggleAuction();
             }
-            if (message.contains("You were spawned in Limbo")) {
+            if (message.contains("You were spawned in Limbo") || message.contains("joined the lobby!")) {
                 try {
-                    Utils.debugLog("Detected in Limbo, stopping everything for 5 minutes");
-                    Utils.addTitle("You got sent to Limbo!");
+                    if (this.lockForJoiningSkyblock.isLocked) return
+                    this.lockForJoiningSkyblock.lock()
```

Comment:
if you have the instance of the lock already, it isn't need to be static
also `this` is not required

Reply:
what is static? never mind, do whatever you think is right, i don't know java at all, and this all i wrote basing on my python experience.